### ISSUE TITLE
Release?

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-industrial/stomp_ros#1]
@jrgnicho: what are the plans with this repository? Should it be released or are we ok with this remaining a source repository only for now?

**Comment by gavanderhoorn** [comment]
Submitted for indexing and devel job in https://github.com/ros/rosdistro/pull/20888.

**Comment by jrgnicho** [comment]
Sorry I went off the grid on this, I've been wondering about separating the stomp_core package  and make it into its own repo as there's been some interest in using it purely as an optimization framework that may or may not be MoveIt related.  In addition to that, it could help with the ROS2 migration as well as the core code may be shared by the MoveIt  and MoveIt 2 versions of stomp_moveit.

**Comment by gavanderhoorn** [comment]
Just to clear up the situation with MoveIt (and not delay that any longer) I would suggest we first get this repository released, make sure the MoveIt integration starts working (again) and then figure out possible refactoring for potential ROS 2 usage.

We started this reshuffling to decouple these packages from `industrial_moveit`. If we now extend that to also take ROS 2 and non-ROS usage into account I fear we'll not finish this any time soon.

**Comment by jrgnicho** [comment]
@gavanderhoorn agreed, lets get this repo release first and then think about those other more ambitions objectives.

**Comment by gavanderhoorn** [comment]
Submitted #2 in preparation for a release.
